j2lint/linter/indenter/node.py

```diff
@@ -120,9 +123,9 @@ def check_indentation(self, result, lines, line_no=0, indent_level=0):
                     result, lines, line_no + 1, indent_level + INDENT_SHIFT)
                 self.check_indent_level(result, node)
                 continue
-            elif node.tag in END_TAGS:
+            if node.tag in END_TAGS:
```

Comment:
I guess pylint may be complaining but I would still keep the `elif` here to help readability - even if I agree that because we get out of each statement it is ok to have `if`. Did you have a warning?

Reply:
yes , I got a warning for using elif with continue

Reply:
ok we can keep it that way - I will add an issue to refactor this method at some point